src/rotation.zig

```diff
@@ -189,9 +196,18 @@ pub const RotationModes = struct {
 			return blocks.meshes.modelIndexStart(block) + @min(block.data, 5);
 		}
 
-		fn rotateZ(data: u16) u16 {
-			var neighbor: Neighbor = @enumFromInt(data);
-			return @intFromEnum(neighbor.rotateZ());
+		fn rotateZ(data: u16, angle: Degrees) u16 {
+			comptime var rotationTable: [24]u8 = undefined;
```

Comment:
Why not use a `[4][6]u8`? This would reduce the access code and make it more safe (less multiplications by hardcoded array length 6):
```diff
- rotationTable[data + @as(u16, @intCast(@intFromEnum(angle)))*6]
+ rotationTable[@intFromEnum(angle)][data]
```

Reply:
No particular reason, except avoiding second dereference, but I am all in for that change.

Reply:
In Zig arrays are flat in memory, the assembly code of both approaches should be identical.